modules/backend/formwidgets/codeeditor/assets/less/codeeditor.less

```diff
@@ -119,4 +120,18 @@
         }
     }
 
+}
+
+#cms-master-tabs {
+	.field-codeeditor {
+		.editor-toolbar {
+			>ul {
+				>li {
+					>a {
+						color: @color_3;
+					}
+				}
+			}
+		}
+	}
 }
```

Comment:
It's not necessary to have this much nesting for only one rule. I would prefer that it was all one line for the selector this case as it's only used for one rule.

Reply:
Please could you check my update and let me know if that's ok or not? I was not sure about spacing the brackets.